drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -103,8 +103,8 @@ def enrollment_access_summaries(filters: nil)
     end
 
     def staff_assignments
-      # n+1, not performant for queries on colletions
-      object.staff_assignments.order(created_at: :desc, id: :desc)
+      # n+1, not performant for queries on collections
+      object.staff_assignments.viewable_by(current_user).order(created_at: :desc, id: :desc)
```

Comment:
Just for posterity here's the link to the conversation when this bug slipped in: https://github.com/greenriver/hmis-warehouse/pull/4578/files#r1697269513 😅